specification/dimensions/skupriceid.md

```diff
@@ -0,0 +1,90 @@
+### SKUPriceID
+
+A unique identifier that defines the price of a SKU that incurred a charge.  This identifier represents a unique collection of nullable attributes that determine the price, as referenced on a price list published by a provider.
+
+The composition of SKU Price attributes may differ from one provider to the next.  For example, the provider may use a Region to define the price, whereas another may use a Region to define the SKU itself.  Examples of SKU Price attributes may include, but are not limited to:
```

Comment:
Would be great if we don't have to mention these attributes and instead just state that things like Tier, Term and payment options can be associated with the SKU or the SKU Price and that the id will be the same as SKU when the unit price is directly attached to the SKU. 

I think you've done a good job of describing here but we've leaned towards avoiding adding a bunch of descriptions into the column definition and instead depended on having details like this in an appendix OR the glossary now that it will be in v1.0.

Reply:
Will put SKU and SKU Price into the glossary and remove this section.